sklearn/metrics/scorer.py

```diff
@@ -98,13 +146,16 @@ def __call__(self, estimator, X, y_true, sample_weight=None):
         super(_PredictScorer, self).__call__(estimator, X, y_true,
                                              sample_weight=sample_weight)
         y_pred = estimator.predict(X)
+        kwargs = self._get_labels_from_estimator(estimator,
```

Comment:
ohh you meant in __call__. yea this might be easier to read later. i'll make the change.